.babelrc-v9.json

```diff
@@ -1,22 +1,9 @@
 {
```

Comment:
## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27313&commitId=87c19802953f46b4ea43e1d02ac984d9a4c1aa37&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1258 screenshots unchanged, 0 screenshots with different dimensions and 2 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox.Checkbox Example Label Flex Column.chromium.png|22| Changed|
|Checkbox.Checkbox Example Label.chromium.png|22| Changed|

</details>


<div id="vrtCommentFluentUI-v0"/>